modules/thermal_hydraulics/src/userobjects/ADNumericalFlux3EqnBase.C

```diff
@@ -13,6 +13,9 @@ InputParameters
 ADNumericalFlux3EqnBase::validParams()
 {
   InputParameters params = GeneralUserObject::validParams();
+  params.addClassDescription(
+      "Base class for retrieving numerical fluxes in the the three-equation model");
```

Comment:
Does it do anything to add class descriptions to un-registered classes like base classes?

Reply:
not really.

Reply:
Doesn't hurt anything, so I'm not asking you to remove, but I was just curious.